test/parsing/errors/commentEOF-detailed.good

```diff
@@ -0,0 +1,5 @@
+─── error in commentEOF.chpl:3 [CommentEOF] ───
+  EOF in comment.
+  Unterminated comment started here:
+  Nested comment started here:
```

Comment:
This looks awkward unless we also print the code.

Reply:
I'll take out the colons but leave these notes with locations. Let me know if that feels overly awkward to you as well

Reply:
My reservation here is that even after we take out the colons, in detailed mode, it'll just say "Unterminated comment started here" without any reference to the location (what's "here"?)

Reply:
On the other hand, with locations it looks like this:
```
─── syntax in test/parsing/errors/commentEOF.chpl:3 [CommentEOF] ───
  End-of-file in comment.
  Unterminated comment started here
      |
    1 | /* this is an unterminated multiline comment
    2 |  /* which contains a nested comment */
      |
  Nested comment started here
      |
    2 |  /* which contains a nested comment */
      |
```